### PR TITLE
Migrate metadata package rules from bazel-toolchains

### PR DESCRIPTION
**Pull request by alex1545**

### CODE REVIEW

**Comment by nlopezgi** [comment]
This PR is migrating code from:
https://github.com/bazelbuild/bazel-toolchains/tree/master/src/go/cmd and
https://github.com/bazelbuild/bazel-toolchains/tree/master/rules/container to this repo.

File: contrib/automatic_container_release/BUILD [hunk 1]
```diff
@@ -11,8 +11,19 @@
...
 
 exports_files(["run_checker.sh.tpl"])
+
+bzl_library(
+    name = "packages_metadata",
+    srcs = [
+        "metadata_merge.bzl",
+        "packages_metadata.bzl",
+    ],
+)
```
Comment (nlopezgi) [inline contrib/automatic_container_release/BUILD]: While you're here, can you add a bzl_library for the other .bzl file in this folder (configs_test)

Comment (alex1545) [reply contrib/automatic_container_release/BUILD]: Done

**Pull request merged**